backend/src/main/kotlin/com/tngtech/apicenter/backend/connector/rest/controller/ServiceController.kt

```diff
@@ -25,12 +29,12 @@ class ServiceController @Autowired constructor(
 
     @PostMapping
     @ResponseStatus(HttpStatus.CREATED)
-    fun uploadSpecification(@RequestBody specificationFileDto: SpecificationFileDto): SpecificationDto {
-        val specification = specificationFileDtoMapper.toDomain(specificationFileDto)
-
-        serviceHandler.addNewSpecification(specification, specification.metadata.id, specificationFileDto.fileUrl)
-
-        return specificationFileDtoMapper.fromDomain(specification)
+    fun uploadSpecifications(@RequestBody specificationFileDtos: List<SpecificationFileDto>): List<Specification> {
```

Comment:
Is this a `List` or a `Set`?

Reply:
It's a list, my rational being it's fundamentally receiving JSON, which only supports ordered arrays and keyed objects. (This signature is how it appears in the documentation.)